gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -479,6 +481,16 @@ def run(argv=None):
     num_partitions = 1
 
   if known_args.output_table:
+    schema_file = tempfile.mkstemp(prefix=known_args.output_table,
+                                   suffix=_BQ_SCHEMA_FILE_SUFFIX)[1]
+    schema = (
+        schema_converter.generate_schema_from_header_fields(
+            header_fields, processed_variant_factory, variant_merger))
+    schema_json = (
+        schema_converter.convert_table_schema_to_json_bq_schema(schema))
+    with filesystems.FileSystems.create(schema_file) as file_to_write:
+      file_to_write.write(schema_json)
```

Comment:
Doesn't a simple `json.dump()` do the same thing? Note this will be run locally on the main monitor machine.

Reply:
Discussed offline. `json.dump()` is used to convert dict to json string, not to write to a file.